force-app/main/default/lwc/mortgage/__tests__/mortgage.test.js

```diff
@@ -0,0 +1,40 @@
+import { getTermOptions, calculateMonthlyPayment } from 'c/mortgage';
+
+describe('c-mortgage', () => {
+    describe('calculateMonthlyPayment function', () => {
+        it('returns monthly payment with valid inputs', () => {
+            const principal = 50000;
+            const years = 20;
+            const rate = 3;
+
+            // Get a two decimal place string representation of long decimal
+            const expectedResult = (277.298798926959).toFixed(2);
+
+            const monthlyPayment = calculateMonthlyPayment(
+                principal,
+                years,
+                rate
+            );
+            // convert long decimal to two decimal places and compare
+            expect(monthlyPayment.toFixed(2)).toBe(expectedResult);
+        });
+
+        it('returns zero with invalid inputs', () => {
+            // each value tests each param as falsy, third param also tested for explicit zero value
+            const invalidInputs = require('./data/invalidInputsTestData.json');
```

Comment:
We should move this outside the test body.

Reply:
This entire data set is only applicable to this test. Moving it outside will make the test less clear.

Reply:
This it. Then it should be moved to `const INVALID_INPUTS` at the top of the test, to make it consistent. Also, does the test cover cases where non-numeric data gets passed to the component (i. e. a String)?

Reply:
The function doesn't implement any specific logic around non-numeric data. This test is to handle the specific line of logic that returns zero when falsy values (or zero in the case of rate) are passed in. 

Since this is a service function, error handling would be handled up the stack (presumably) as it was not implemented here.